src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -140,13 +140,22 @@ public void InitializeService(ServiceHost serviceHost)
                     Version = Version.Parse(serverInfo.ServerVersion),
                     Platform = hostInfo.Platform
                 };
-
                 var db = SqlAssessmentService.GetDatabaseLocator(server, connection.Database);
-                var connectionString = ConnectionService.BuildConnectionString(connInfo.ConnectionDetails);
-                var results = await GetAssessmentItems(server, connectionString);
-                var result = new MigrationAssessmentResult();
-                result.Items.AddRange(results);
-                await requestContext.SendResult(result);
+                var connectionStrings = new List<string>();
+                if (parameters.Databases != null) 
+                {
```

Comment:
Do you really need to check for Databases not being null here? If the contract says they shouldn't be null then typically it should be fine to assume it's not null